src/nodetreeview.cpp

```diff
@@ -602,7 +602,7 @@ void NodeTreeView::mousePressEvent(QMouseEvent *event)
             case NodeItem::Type::TagItem: {
                 auto oldIndexes = selectionModel()->selectedIndexes();
                 for (const auto &ix : qAsConst(oldIndexes)) {
-                    auto itemType =
+                    itemType =
```

Comment:
I think this arbitrary variable re-use is confusing. This should be a new variable since it has little to do with the previous one. Maybe in the future, we end up changing the code such that the original `itemType` is still used beyond this scope, and then it may be changed to something random only in this case, which can be easy to miss.

If you really want to avoid shadowing warnings, the fix is to choose different names in my opinion.

Reply:
Nice catch, and I completely agree. I renamed it to `selectedItemType`, which seems more accurate.